mps-model-adapters/src/main/kotlin/org/modelix/model/mpsadapters/MPSNode.kt

```diff
@@ -129,9 +129,9 @@ data class MPSNode(val node: SNode) : IDeprecatedNodeDefaults {
     }
 
     override fun setReferenceTarget(link: IReferenceLink, target: INode?) {
-        val ref = node.references.first { MPSReferenceLink(it.link).getUID() == link.getUID() }
```

Comment:
Should we add a test for this?

Reply:
This is covered in the extended test solution. That's how I discovered it in the first place.